core/src/main/python/discover.py

```diff
@@ -224,8 +245,17 @@ def __discover(model_context, aliases, credential_injector, helper):
                                                         ae.getLocalizedMessage(), error=ae)
         __logger.throwing(ex, class_name=_class_name, method_name=_method_name)
         raise ex
-
     __disconnect_domain(helper)
+
+    if model_context.is_remote():
+        print 'Discovered Remote Domain - showing files to collect for archive'
```

Comment:
Why are you using "print" instead of __logger?

Reply:
It was requested by Robert to not mix in with Summary. As a separate piece of stdout

Reply:
In fact, you cannot use the logger.  Otherwise, it gets mixed up with all other log messages.  The Summary Logger only supports Warning/Error levels so we cannot use that either.